styles/weather.css

```diff
@@ -0,0 +1,3 @@
+.container .row div .weather-button{
```

Comment:
If you put all of the weather widget inside a div with the class `weather` then you can change this selector to:
`.weather button[type="submit"]`